regions/io/crtf/write.py

```diff
@@ -51,6 +50,8 @@ def write_crtf(regions, filename, coordsys='fk5', fmt='.6f', radunit='deg'):
     """
     Converts a `list` of `~regions.Region` to CRTF string and write to file.
 
+    See :ref:`gs-crtf`
```

Comment:
this won't show up when one looks at the docstring from the python terminal, but I think that's OK.  These methods are not complicated, having the example only in the narrative docs should be sufficient.